database/buildSchema/34_b_verification.sql

```diff
@@ -0,0 +1,21 @@
+-- verification
+CREATE TABLE public.verification (
+    id serial PRIMARY KEY,
+    unique_id varchar UNIQUE NOT NULL,
+    application_id integer REFERENCES public.application (id) NOT NULL,
+    code varchar,
+    message varchar,
+    data jsonb,
+    created_time timestamptz DEFAULT NOW(),
+    expiry_time timestamptz,
+    is_verified boolean DEFAULT FALSE,
+    TRIGGER public.trigger
+);
+
+-- TRIGGER (Listener) on verification table
+CREATE TRIGGER verification_trigger
+    AFTER INSERT OR UPDATE OF trigger ON public.verification
+    FOR EACH ROW
+    WHEN (NEW.trigger IS NOT NULL AND NEW.trigger <> 'PROCESSING' AND NEW.trigger <> 'ERROR')
+    EXECUTE FUNCTION public.add_event_to_trigger_queue ();
+
```

Comment:
Wondering if we need to list the tables that have the TRIGGER column. As ones that need to be accounted on row_level_permission. There was something about allowing these to be changed by front-end that I can't remember what it was.

Reply:
Yeah, I should update the "Triggers & Actions" documentation to include this. Will do.

Reply:
Okay, have updated the list of trigger tables in docs: 1c31ee5c

Reply:
Thanks. That's great